plugins/modules/cloud/misc/proxmox.py

```diff
@@ -163,13 +163,11 @@
       - Various module options used to have default values. This cause problems when
         user expects different behavior from proxmox by default or fill options which cause
         problems when they have been set.
-      - The default value is C(compatibility), which will ensure that the default values
+      - The default value is C(no_defaults), which will ensure that no default values
         are used when the values are not explicitly specified by the user.
-      - From community.general 4.0.0 on, the default value will switch to C(no_defaults). To avoid
```

Comment:
I would keep something like "The default switched from C(compatibility) to C(no_defaults) in community.genral 4.0.0.".

Reply:
Added in https://github.com/ansible-collections/community.general/pull/2937/commits/9e23db2e49dd03a01e09ea75d3c72c91f9f23682.